api/v1alpha1/cluster_promotion_task_types.go

```diff
@@ -0,0 +1,27 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +kubebuilder:resource:scope=Cluster,shortName={clusterpromotask,clusterpromotasks}
+// +kubebuilder:object:root=true
+// +kubebuilder:printcolumn:name=Age,type=date,JSONPath=`.metadata.creationTimestamp`
+
+type ClusterPromotionTask struct {
```

Comment:
While I know ClusterPromotionTask is technically accurate, I wonder if we want to term this "GlobalPromotionTask". We already refer to credentials and ServiceAccounts that are to be used in a global capacity as "global" credentials and global serviceaccounts, and this would be along those lines.

The "cluster" is less meaningful in